boards/common/esp8266/Kconfig

```
@@ -15,3 +15,13 @@ config BOARD_COMMON_ESP8266
     select HAS_PERIPH_SPI
     select HAS_PERIPH_UART
     select HAS_PERIPH_UART_MODECFG
+
+    select HAVE_SAUL_GPIO
+
+config MODULE_BOARDS_COMMON_ESP8266
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_ESP8266
```

```suggestion
    depends on BOARD_COMMON_ESP8266
    depends on HAS_ARCH_ESP8266
```
Doesn't it require `BOARD_COMMON_ESP8266`. I'm just wondering because this is different to the definition for ESP32.

---

I can add it, sure. Keep in mind that the only way for this file to be loaded is that we are on an esp8266 board, otherwise the symbol is not there. For consistency it makes sense to add it.

---

Changed the dependency in 9e66f139cc25e71ff308710012fbcccd6893e842